### ISSUE TITLE
Issue with converting Gregorian date to Jalali

### ISSUE THREAD

**Issue by mobinzk** [samanzamani/PersianDate#19]
When I try to convert this specific date 2018/12/15 (Year/Month/Day) I get crash saying `arrayindexoutofboundsexception`.

Here is an example that I get crash with.
```
PersianDate pdate = new PersianDate().initGrgDate(2018,12,15);
PersianDateFormat pdformater = new PersianDateFormat("Y/m/d");
String dd = pdformater.format(pdate);
```

Could you please advise me on this please?

**Comment by samanzamani** [comment]
Thank you for your report.
I solved this issue in version 0.6

**Comment by mobinzk** [comment]
Thanks for your rapid respond and to resolve the issue.
